CGFloatType/CGFloatType.h

```diff
@@ -37,6 +37,9 @@ CGFloat CGFloatWithDouble(double value);
 /// Returns a CGFloat from a float
 CGFloat CGFloatWithFloat(float value);
 
+#pragma mark - CGFloat comparison
+
+extern BOOL CGFloatEquals(CGFloat x, CGFloat y);
```

Comment:
Do we need the `extern` here, I though it doesn't have any effect in a framework header?